.gitignore

```diff
@@ -44,3 +44,6 @@ Singularityfile
 #whitelist
 !reports/**/*.csv
 !reports/**/*.png
+
+catboost_info/*
+plot/*
```

Comment:
```suggestion
```

To avoid setting a new precedent, the recommended way for all those 3rd party artifacts is to put them under flaml/setup or flaml/lib depending if they're needed only during setup or also at runtime.
Those 2 folders are already excluded in git.
Besides, it standardizes the folder structure, making each framework easier to understand and maintain.

If those folders are generated at runtime: you need to find a way to redirect the output either:
- in a tmp folder.
- somewhere in the `config.output_dir`.